src/environmentd/tests/server.rs

```diff
@@ -1997,7 +1997,6 @@ async fn test_concurrent_id_reuse() {
 // TODO: re-enable once we figure out how to do this test without
 // relying on internal resources:
 // <https://github.com/MaterializeInc/materialize/issues/25294>
-#[ignore]
```

Comment:
you also want to remove the lines above this!

Reply:
:kalispera: